crates/storage/provider/src/providers/mod.rs

```diff
@@ -667,7 +667,7 @@ where
 
         if let Some(block) = self.tree.pending_block_num_hash() {
             if let Ok(pending) = self.tree.pending_state_provider(block.hash) {
-                return self.pending_with_provider(pending)
+                return self.pending_with_provider(pending);
```

Comment:
please revert these unrelated changes. use the nightly formatter for reth `cargo +nightly fmt`. you can also use shortcut `make lint`.